eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -102,6 +103,13 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.executeUpdate("DROP TABLE audit.network_fees")
       }
 
+      def migration1011(statement: Statement): Unit = {
+        statement.executeUpdate("ALTER TABLE audit.path_finding_metrics ADD COLUMN payment_hash TEXT")
+        statement.executeUpdate("ALTER TABLE audit.path_finding_metrics ADD COLUMN routing_hints JSONB")
```

Comment:
The schema after the migration must be exactly the same as without migration, that's the goal of those `migrationXY` methods. Otherwise we have to think of every possible combination in the queries it will be become quickly unmangeable over time. Here the columns can be nullable or mandatory depending on history.

For this particular case, I think making `routing_hints` nullable is an entirely valid choice.

Reply:
Done, I've made the two new columns nullable

Reply:
Just to rephrase my previous comment and stress why it's really important: if the migration methods don't produce the exact same schema as the init script, our linear db schema history becomes a tree.